docs/themes/sphinx_rtd_theme/__init__.py

```diff
@@ -31,33 +31,47 @@ def config_initiated(app, config):
             _('The canonical_url option is deprecated, use the html_baseurl option from Sphinx instead.')
         )
 
+
+def extend_html_context(app, pagename, templatename, context, doctree):
+     # Add ``sphinx_version_info`` tuple for use in Jinja templates
+     context['sphinx_version_info'] = sphinx_version
+
+
 # See http://www.sphinx-doc.org/en/stable/theming.html#distribute-your-theme-as-a-python-package
 def setup(app):
     if python_version[0] < 3:
-        logger.warning("Python 2 is deprecated with sphinx_rtd_theme, update to Python 3")
-    app.require_sphinx('1.6')
-    if sphinx_version <= (2, 0, 0):
-        logger.warning("Sphinx 1.x is deprecated with sphinx_rtd_theme, update to Sphinx 2.x or greater")
-        if not app.config.html_experimental_html5_writer:
-            logger.warning("'html4_writer' is deprecated with sphinx_rtd_theme")
-    else:
-        if app.config.html4_writer:
-            logger.warning("'html4_writer' is deprecated with sphinx_rtd_theme")
+        logger.error("Python 2 is not supported with sphinx_rtd_theme, update to Python 3.")
+
+    app.require_sphinx('5.0')
+    if app.config.html4_writer:
+        logger.error("'html4_writer' is not supported with sphinx_rtd_theme.")
```

Comment:
1. the URL is outdated, resolves to 404;
2. https://github.com/readthedocs/sphinx_rtd_theme diverges greatly, including giving back Dockerfile option for local compiling. So it's not really using `__version__ = '2.0.1alpha1'`

Reply:
This is a direct pull off their git repo for RTD 2.0.0, no other changes were made.